src/style.css

```
@@ -1,3 +1,77 @@
+html {
+  --color-active: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  color: #000;
+}
+
+div,
+li,
+p,
+ul {
+  box-sizing: border-box;
```

stil style property should be use for all elements in the project so should be used like this 
```suggestion
* {
    box-sizing: border-box;
  }
```